### PR TITLE
Tracer: Add a permissive mode.

### PR DESCRIPTION
**Pull request by ltfish**

### CODE REVIEW

**Comment by ltfish** [comment]
Oops it breaks driller. I'll get it fixed.

File: angr/exploration_techniques/tracer.py [hunk 1]
```diff
@@ -195,7 +281,9 @@ def _update_state_tracking(self, state: 'angr.SimState'):
                 if self._compare_addr(self._trace[idx], addr):
                     idx += 1
                 else:
-                    raise Exception('BUG! Please investigate the claim in the comment above me')
```
Comment (rhelmot) [inline angr/exploration_techniques/tracer.py]: did you ever see any instances of this being tripped and it being an actual desync and not a unicorn bug?

Comment (ltfish) [reply angr/exploration_techniques/tracer.py]: Missing, incomplete, or incorrect SimProcedure implementations.

**Comment by rhelmot** [comment]
:(

**Pull request merged**